File: backend/src/AnimeShop/API/Security.hs
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE TypeOperators #-}
module AnimeShop.API.Security (SecurityAPI, securityServer) where

import AnimeShop.Prelude
import AnimeShop.API.Security.EmployeeEntries
import AnimeShop.API.Security.TransportEntries

type SecurityAPI = "security" :>
  (EmployeeEntriesAPI :<|> TransportEntriesAPI)

securityServer :: ServerT SecurityAPI App
securityServer = employeeEntriesServer :<|> transportEntriesServer

File: frontend/src/components/Pages/Security.js
import React from "react";
import EmployeeEntries from "../Security/EmployeeEntries";
import TransportEntries from "../Security/TransportEntries";

export function Security() {
    return (
      <div>
        <EmployeeEntries/>
        <TransportEntries/>
      </div>
    );
  }
